subsys/bluetooth/controller/ll_sw/lll_adv.h

```diff
@@ -73,8 +75,8 @@ struct lll_adv_sync {
 	struct lll_hdr hdr;
 	struct lll_adv *adv;
 
-	uint8_t access_addr[4];
-	uint8_t crc_init[3];
+	uint8_t access_addr[PDU_ACCESS_ADDR_SIZE];
+	uint8_t crc_init[PDU_CRC_SIZE];
```

Comment:
This is not really the CRC size in PDU, but the initial value. Use  different name close to one used in Spec documents?

Reply:
Sure, could you tell me which spec document you're referring to?